### PR TITLE
Fix syntax

### PR DESCRIPTION
**Pull request by zakariaGatter**
get rid of extra prosses 'cat' and use builtin bash commands fix some bash syntax (using sh syntax in a bash script)

### CODE REVIEW

File: xbps-src [hunk 1]
```diff
@@ -9,7 +9,9 @@ print_cross_targets() {
 }
 
 usage() {
-    cat << _EOF
+while read ; do
+    printf "%s\n" "$REPLY"
+done <<- _EOF
```
Comment (sgn) [inline xbps-src:14]: `cat` is fine here, beside a short and rare usage of `usage`, `cat` is easier to reason about.

**Review by sgn — COMMENTED** [review]
In general, I don't like this change.

File: xbps-src (hunk 1 above)
Comment (zakariaGatter) [reply xbps-src:14]: why using something outside `bash` if it have the solution

**Comment by Chocimier** [comment]
Nothing's broken, those are just style adjustments.

**Comment by zakariaGatter** [comment]
> Nothing's broken, those are just style adjustments.

that what i said on my pull request description

**Pull request closed without merging**